andela/tembea
# #161929099 Setup database migrations and models

#### What does this PR do?
Setup the model and migration files

#### Description of Task to be completed?
> - Create the user migration and model
> - Create the trip request migration and model
> - Create the location migration and model
> - Create the address migration and model
> - Create the cab migration and model
> - Create the departments' migration and model

#### How should this be manually tested?
> - Clone the repo to your machine
> - Run `yarn install` ensure you have yarn installed
> - Create your database and ensure you have set up your `.env` using the sample provided.
> - Run `npm run db:migrate` to create the tables
> - Run `npm run db:rollback` to drop them

#### Any background context you want to provide?
This is needed for the initial setup of the database

#### What are the relevant pivotal tracker stories?
#161929099

#### Screenshots (if appropriate)
N/A

#### Questions:
